test/tables/sortable-table.test.js

```diff
@@ -367,3 +367,12 @@ test('invalid arbitrary props filtered out', () => {
   )
   expect(wrapper.find('table').first().prop('invalidProp')).toBe(undefined)
 })
+
+test('passes extra props to cells', () => {
```

Comment:
Can you add a test that confirms invalid extra props don't get passed? E.g., `customCellClass`